lib/miq_automation_engine/engine/miq_ae_engine/drb_remote_invoker.rb

```diff
@@ -110,6 +112,7 @@ class AutomateMethodException < StandardError
 
   DRbObject.send(:undef_method, :inspect)
   DRbObject.send(:undef_method, :id) if DRbObject.respond_to?(:id)
+  LINE_HASH = YAML.load(LINE_HASH_YAML)
```

Comment:
I don't think we need to pay the penalty for this on every invocation...it can probably just be called directly in `get_file_info` when needed.  I would expect this to only happen during Exceptions, which themselves would be rare.